pkg/cmd/roachtest/option/node_list_option.go

```diff
@@ -85,11 +85,14 @@ func (n NodeListOption) SeededRandList(rand *rand.Rand, size int) (NodeListOptio
 
 	nodes := make(map[int]struct{}, size)
 	for range size {
-		node := n[rand.Intn(len(n))]
-		for _, ok := nodes[node]; ok; {
-			node = n[rand.Intn(len(n))]
+		for {
```

Comment:
This function seems more complicated than it needs to be. Isn't the entire function equivalent to the following?

```go
nodes := append([]int{}, n...)
rand.Shuffle(len(nodes), func(i, j int) { nodes[i], nodes[j] = nodes[j], nodes[i] })
return nodes[:size]
```

Reply:
I'd say so. Didn't know about rand.shuffle, will fix